html/test.html

```diff
@@ -1 +1,11 @@
-
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" />
+    <title>Feature Name</title>
+    <link rel="stylesheet" href="/css/test.css" />
```

Comment:
Make sure to add the other stylesheet `style.css` or half the styling will break.

Reply:
Good catch, thanks. Code has been updated.